packages/core/src/lib/core/initialise-lists.ts

```diff
@@ -154,8 +154,8 @@ function getIsEnabled (listKey: string, listConfig: KeystoneConfig['lists'][stri
       create: notOmit,
       update: notOmit,
       delete: notOmit,
-      filter: notOmit && defaultIsFilterable,
-      orderBy: notOmit && defaultIsOrderable,
+      filter: notOmit ? defaultIsFilterable : false,
+      orderBy: notOmit ? defaultIsOrderable : false,
```

Comment:
This was confusing, `&&` technically worked, but, the point is that it is `false` or a function